src/App.js

```diff
@@ -1,5 +1,123 @@
+import Background from "./assets/Background.svg";
+import Logo from "./assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
+
+async function post(url, body) {
```

Comment:
You're creating this function multiple times in separate files. It's better to create a new file for things related to requests, and export the functions from there. Like that you can always import it from a single place in all the places you need it.

Reply:
You're right! Thanks.